package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "rto_do_list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Kindly update your npm scripts to add basic commands for building Webpack in development mode, and webpack dev server. it should be something like this:
```JSON
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "webpack serve --open",
    "build": "webpack"
  }
```